less_3/3_1.py

```diff
@@ -0,0 +1,10 @@
+def double_list(array: list[int]) -> list[int]:
+    res = set()
+    for el in array:

```

Comment:
Чтобы уменьшить итерации
```suggestion
    for el in set(array):

```